deploy/olm-catalog/argocd-operator/0.9.1/argoproj.io_argocds.yaml

```diff
@@ -1672,7 +1672,7 @@ spec:
                       type: object
                     type: array
                   execTimeout:
-                    description: ExecTimeout specifies the timeout in seconds for
+                    description: ExecTimeout specifies the timeout in minutes for
```

Comment:
It can be seconds, minutes or hours based on the suffix user provides. This description needs a change.